impeller/renderer/command.h

```diff
@@ -153,17 +153,17 @@ struct Command {
   bool BindResource(ShaderStage stage,
                     const ShaderUniformSlot& slot,
                     const ShaderMetadata& metadata,
-                    BufferView view);
```

Comment:
These are potentially big (BufferView has a shared_ptr in it too).

Reply:
Buffer view copies don't copy the buffer. They just have a pointer to the buffer.